guardrails/src/drivers/rest/routers/admin.py

```
@@ -0,0 +1,273 @@
+        description=version.description,
+        created_at=version.created_at,
+    )
+
+
+@router.get(
+    "/{domain_id}/configs/{config_id}/versions",
+    response_model=List[VersionResponse],
+    tags=["versions"],
+)
+async def list_versions(
+    domain_id: UUID,
```

Ruff (0.14.8)

Unused function argument: `domain_id`

(ARG001)